test/cell_test.rb

```diff
@@ -59,5 +59,17 @@ def test_fire_upon_lowers_ship_health_by_one
     @cell.place_ship(@submarine)
     @cell.fire_upon
     assert_equal 1, @cell.ship.health
+    assert_equal 2, @cell.ship.health
```

Comment:
I don't think this line is needed, and I don't think it would pass; the ship's health should be "1" after line 60, and wouldn't go back up to 2. Let me know if you had another reason for adding it that I'm missing.